photon-api/src/main/scala/com/linkedin/photon/ml/algorithm/RandomEffectCoordinate.scala

```diff
@@ -138,7 +149,6 @@ object RandomEffectCoordinate {
    *
    * @note The score is the dot product of the model coefficients with the feature values (in particular, does not go
    *       through non-linear link function in logistic regression!).
-   *
```

Comment:
I agree - Scala is doing this on its own, and I can't find an option in the style to make it stop. I think we should leave it be for now, until we can find a way to lock it in the style.